app-admin/passwordsafe/passwordsafe-1.08_beta.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit eutils flag-o-matic wxwidgets cmake-utils
```

Comment:
as far as I can see you don't need `eutils` here, you just use functions from `desktop.eclass`